### PR TITLE
RB-1527: Skip button does not work in chat agent when a 'yes' or 'no' question is asked

### PR DESCRIPTION
**Pull request by sidhuko**
https://rainbird.atlassian.net/browse/RB-1527

### CODE REVIEW

File: public/chirpy.js [hunk 1]
```diff
@@ -450,9 +450,9 @@ function addQuestion (question) {
     resizeAndScroll();
     d3.select('#sendButton')
         .on('click', function() {
-            if (checkInputAndHighlightButtons(question)) {
-                send(question);
-            }
+            // @todo add missing skip;
```
Comment (LawrieR) [inline public/chirpy.js]: Should this comment still be here?

Comment (sidhuko) [reply public/chirpy.js]: Good spot, thanks!

File: public/chirpy.js [hunk 2]
```diff
@@ -490,15 +489,23 @@ function send(question, input) {
...
-            object: question.type === 'Second Form Object' ? null : question.object,
+        const data = {
             cf: 100,
-            unanswered: true
-        });
+            unanswered: true,
+            relationship: question.relationship,
+        }
+
+        if (question.type === 'Second Form Subject') {
```
Comment (LawrieR) [inline public/chirpy.js]: Sorry I didn't spot this before but it looks like we've dropped skipping second form object questions? If it's "second form object" we'd send back a response without a subject and I think we reject it in that instance.

Comment (sidhuko) [reply public/chirpy.js]: I see, wasn't called in my example, it's back now.

**Review by LawrieR — APPROVED** [review]

**Pull request merged**